src/main/webapp/dataverseuser.xhtml

```diff
@@ -29,6 +29,7 @@
                     <f:viewParam name="editMode" value="#{DataverseUserPage.editMode}"/>
                     <f:viewParam name="redirectPage" value="#{DataverseUserPage.redirectPage}"/>
                     <f:viewParam name="selectTab" value="#{DataverseUserPage.selectTab}"/>
+                    <f:viewAction action="#{dataverseSession.updateLocaleInViewRoot}"/>
```

Comment:
There may be a way to do this in a way that does not require calling this method from every page; but as I said, I wasn't able to find one. Calling this method from, or otherwise achieving this by doing something in the dataverse_template seems logical (would solve having to call an extra method from every page...). But the very problem at hand appears to be that no matter what we do in that template, it only takes effect after the page's own init() method is executed. (the method called from the v:fiewAction in individual pages does in fact replicate something we are already doing in the template...)

Reply:
Ah I see what you meant now, I was confused when I read through the details earlier. Sounds good!